connector_easypost_tracker/models/stock_picking_tracking_group.py

```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 import logging
-from openerp import models, fields
+from openerp import fields, models
```

Comment:
It's funny how much I hated the menial task of alphabetizing imports, but now that I'm used to it I can't live without them alphabetized. It's one of those can't unsee things, right!? 😆